test/ellipsis.js

```diff
@@ -1,7 +1,10 @@
 describe('Handlebars.helpers.ellipsis', function() {
 
     var tests = [
-        { text: 'The quick brown fox', len: 12, expected: 'The quick...' }
+        { text: '', len: 1, expected: undefined },
```

Comment:
Yeah, you're completely right - it makes sense that if you pass an empty string you should get a string back. It's just a matter of changing [this line](https://github.com/marianosimone/duckduckgo-template-helpers/blob/fix-ellipsis/template_helpers.js#L205) to `return "";`.

Would you like to make the change it or shall I do it?